flocker/provision/_install.py

```diff
@@ -605,9 +613,46 @@ def configure_zfs(node, variants):
     ])
 
 
+def _uninstall_flocker_ubuntu1404():
+    # TODO Does a real upgrade (instead of uninstall/install) work?
+    return run_from_args([
+        b"apt-get", b"remove", b"-y", b"--purge", b"clusterhq-python-flocker",
```

Comment:
If this purges `/var/lib/flocker` it's also going to delete cluster config... do you really want that?